docs/src/content/docs/guides/components.mdx

```diff
@@ -28,12 +30,12 @@ import AnotherComponent from '../../components/AnotherComponent.astro';
 </AnotherComponent>
 ```
 
-Because Starlight is powered by Astro, you can use components built with any UI framework in your MDX files.
+Because Starlight is powered by Astro, you also can use components built with any [supported UI framework (React, Preact, Svelte, Vue, Solid, Lit and Alpine)](https://docs.astro.build/en/core-concepts/framework-components/) in your MDX files.
```

Comment:
Not clear here what the "also" would refer to. I'm assuming Astro components but we don't actually mention those so maybe we remove?

```suggestion
Because Starlight is powered by Astro, you can use components built with any [supported UI framework (React, Preact, Svelte, Vue, Solid, Lit, and Alpine)](https://docs.astro.build/en/core-concepts/framework-components/) in your MDX files.
```

Alternatively, I guess we could rework to mention Astro components too.

One thing I wasn't sure of was how much to say about adding framework integrations. Maybe we need to say this is supported _if_ you add an integration?

Reply:
Updated your comment to use the same wording we use for Markdoc! "You can add support for..."